pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -11,10 +11,13 @@
     <ProjectReference Include="..\Microsoft.NETCore.DotNetHost\Microsoft.NETCore.DotNetHostPolicy.pkgproj">
       <TargetFramework>.NETCoreApp1.0</TargetFramework>
     </ProjectReference>
+    <ProjectReference Include="..\Microsoft.NETCore.DotNetHost\Microsoft.NETCore.DotNetHostPolicy.pkgproj">
+      <TargetFramework>.NETCoreApp1.1</TargetFramework>
+    </ProjectReference>
   </ItemGroup>
-
+  
   <Target Name="ReadDependenciesFromProjectJsonFile" BeforeTargets="GetPackageDependencies">
-    <ReadLinesFromFile File="$(MSBuildThisFileDirectory)project.json">
+    <ReadLinesFromFile File="$(MSBuildThisFileDirectory)\netcoreapp1.0\project.json">
```

Comment:
Shouldnt this depend upon the target framework as opposed to hardcoded to netcoreapp1.0?